packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -1461,6 +1577,610 @@ const Conversation = WebexPlugin.extend({
     });
   },
 
+
+  /**
+   * common interface for facade of generator functions
+   * @typedef {object} IGeneratorResponse
+   * @param {boolean} done whether there is more to fetch
+   * @param {any} value the value yielded or returned by generator
+   */
+
+  /**
+   * @param {object} options
+   * @param {string} options.conversationId
+   * @param {string} options.conversationUrl,
+   * @param {?string} options.minActivities how many activities to return in first batch
+   * @param {?string} [options.queryType] one of older, newer, mid. defines which direction to fetch
+   * @param {?object} [options.search] server activity to use as search middle date
+   *
+   * @returns {object}
+   * returns three functions:
+   *
+   * getOlder - gets older activities than oldest fetched
+   *
+   * getNewer - gets newer activities than newest fetched
+   *
+   * jumpToActivity - gets searched-for activity and surrounding activities
+   */
+  listActivitiesThreadOrdered(options) {
```

Comment:
Can you keep this consistent with other functions and have them be 2 separate arguments?
```suggestion
  listActivitiesThreadOrdered(conversationUrl, activityId) {
```

Reply:
This function accepts more arguments than the two that get destructured at the top: the `baseOptions` variable spreads the options object (minus the destructured properties) and is passed along to the private method